src/guides/v2.3/frontend-dev-guide/layouts/layout-types.md

```diff
@@ -91,6 +91,11 @@ Sample page layout declaration file: `<Magento_Theme_module_dir>/view/frontend/l
 
 Use the `layout` attribute in the `page` node of a page configuration file to define a layout type for the page. The following example shows how to use the `3 columns` page layout type for the [Wish List Sharing]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Wishlist/view/frontend/layout/wishlist_index_share.xml#L8) page:
 
+Override the default `wishlist_index_share.xml` in any one of the following paths and add the `layout="3columns"` in the `page` node.
+
+*  Override the layout in custom theme: `<custom_theme_dir>/Magento_Wishlist/layout/wishlist_index_share.xml`
```

Comment:
Hi, @ajithkumar-maragathavel !

Good PR. However, I have a question from the reader's side. Could you please add a small clarification in which cases I should use `module override` option and in which ones I should use `theme` override?

Thank you!

Reply:
@atwixfirster Thanks for the suggestions. I have updated the PR with the explanation. Can you please review it and share your suggestion?

Reply:
Hi, @ajithkumar-maragathavel !

Thank you for the adding explanations. I would like to move them into each point.

Please check my vision and let me know if it makes sense :

```
*  Override the layout in custom theme (_in case when a custom-built theme applied on the storefront_): `<custom_theme_dir>/Magento_Wishlist/layout/wishlist_index_share.xml`
 *  Override the layout in custom module (_in case of building third-party extensions and need to make changes to the existing layout_): `<custom_module_dir>/view/frontend/layout/wishlist_index_share.xml`
```

Thank you!

Reply:
Hi, @atwixfirster Your version looks pretty good. I have updated the PR. Thanks

Reply:
Thank you, @ajithkumar-maragathavel!